hatchways/team-cosmic-crisp
# FS Reviews

### What this PR does (required):
- User can view all reviews for a sitter
- User can create a review for sitter only when logged in.
- Added pagination for reviews

### Screenshots / Videos (required):

![1](https://user-images.githubusercontent.com/75184965/121300902-46a3c480-c8ac-11eb-834b-e8a2718aacd7.PNG)

![2](https://user-images.githubusercontent.com/75184965/121300913-4a374b80-c8ac-11eb-81a1-d1a9ae70b49b.PNG)

![3](https://user-images.githubusercontent.com/75184965/121300922-4e636900-c8ac-11eb-8009-e31dab9c2167.PNG)

### Any information needed to test this feature (required):
- `npm start` on client | `npm run dev` on server
-  login using existing account or sign up for an account
- click on the available sitter profiles to check their reviews
- click on 'write a review' to submit a review for respective sitter

### Any issues with the current functionality (optional):
- None
